src/main/drivers/sound_beeper.h

```diff
@@ -30,7 +30,7 @@
 #endif
 
 typedef struct beeperConfig_s {
-    ioTag_t ioTag;
+    ioTag_t tag;
```

Comment:
it is certainly more descriptive, I changed it to tag as so many others are tag, after this is finalised we can go through and do a big rename of them all.